src/Pages/login/login.jsx

```diff
@@ -12,50 +13,51 @@ export default () => {
   const [userNameAlert, setUserNameAlert] = useState(null);
   const [passwordAlert, setPasswordAlert] = useState(null);
 
-  function checkUsername () {
+  const [isInputsValid, setInputsValid] = useState(false);
+
+  const [isLoginClicked, setLoginClicked] = useState(false);
```

Comment:
We shouldn't leave spaces in-between these. Code that is similar (in this case the component's state variables) can be in a group.

Reply:
Fixed in 925d48b0a98fb7bd42aa97c5a80d13fa92be9888